src/main/resources/db/changelog/changes/01-create-books-table.yaml

```
@@ -0,0 +1,48 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-books-table
+      author: Mykola Skrypalov
+      changes:
+        - createTable:
+            tableName: books
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false

```

`primaryKey` already has `nullable false` inside it
```suggestion
                    primaryKey: true
```